lib/clusters.py

```diff
@@ -86,6 +86,7 @@ def merge_with(self, other) -> None:
       print(f"Newly merged cluster {self.orders} manual override unset.")
       self.manual_override = False
     self.non_reimbursed_trackings.update(other.non_reimbursed_trackings)
+    self.reconcile = self.reconcile or other.reconcile
```

Comment:
does this play weirdly with Python's odd truthiness? I honestly don't know

Reply:
Fortunately truthiness doesn't come into play here; these are both actual honest-to-God booleans in all cases.

Reply:
I made it bullet-proof just in case ... stupid Pyhton.

Reply:
This is moot anyway because there's nothing to add here.  I didn't add this attribute to clusters, only to trackings, and trackings never get merged.  I've removed this.